src/lib/utils/create-expiration.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Creates a `Date` object that is a certain number of milliseconds in the future.
+ * @param milliseconds the number of milliseconds before the expiration date
+ */
+export const createExpiration = (milliseconds: number) => {
+  return new Date(Math.floor(new Date().valueOf() / 10) * 10 + milliseconds);
```

Comment:
This was added to round the expiration date to the nearest 100th. This is because when testing the tested method uses this same function to check if the generation occurred as expected and depending on how fast your computer is and how close it was to going to the next 1ms the time may be different when it runs the code in the store and when it compares to the test so this buffers it so that as long as it is was in the same 100th of a second the test will pass.

I figured `0.01s` is close enough for expiring.

Reply:
Got it - makes sense to me.
Maybe consider adding a comment though... It may be non-obvious to a developer-user what's going on here, if they aren't thinking about testing.

Reply:
Done.